docs/geedocs/5.2.3/answer/7160003.html

```diff
@@ -52,8 +54,384 @@ <h5>
       <li>Ubuntu 14.04 LTS and 16.04 LTS</li>
     </ul>
     <p>Google Earth Enterprise 5.2.3 is compatible with Google Earth Enterprise Client (EC) version 7.1.5 and above.</p>
-    <p>If you're upgrading from GEE 5.1.3, you may have to reinstall the prerequisite libraries after uninstalling 5.1.3 but before installing 5.2.3.</p>
+    <p><p>To upgrade from Open GEE 5.2.0, do NOT uninstall it. We recommend that you upgrade Open GEE 5.2.0 by simply installing Open GEE 5.2.3. Installing Open GEE 5.2.3 on top of Open GEE 5.2.0 will ensure that your PostgreSQL databases are backed up and upgraded correctly to the new PostgreSQL version used by Open GEE 5.2.3.</p></p>
   <h5>
+    <h5>
+      <p id="resolved_issues_5.2.3">Resolved Issues</p>
+    </h5>
+    <div>
+      <table class="nice-table">
+        <tbody>
+          <tr>
+            <th>Number</th>
+            <th class="narrow">Description</th>
+            <th>Resolution</th>
+          </tr>
+          <tr>
+            <td>225</td>
+            <td>Fusion lets you create folder with space in the name</td>
+            <td>AssetManager UI only allows letters, numbers, and underscores for directory name creation</td>
+          </tr>
+          <tr>
+            <td>337</td>
+            <td>Add a compiler version check to scons scripts</td>
+            <td>Scons scripts ensure gcc version 4.8+ is installed before continuing with builds</td>
+          </tr>
+          <tr>
+            <td>498</td>
+            <td>Add option to geconfigureassetroot to delete a volume</td>
+            <td>--removevolume option added for geconfigureassetroot</td>
+          </tr>
+          <tr>
+            <td>535</td>
+            <td>DownloadTutorial.sh often is not staged properly for install</td>
+            <td>Tutorial data is included in the Fusion RPM</td>
```

Comment:
I'm not sure if this is accurate. I thought it would only include download_tutorial.sh. But, I suppose if the tutorial data is staged, then it might include it. We should double check that, but we can do that as a follow-up